app/networkapi/settings.py

```diff
@@ -130,8 +131,8 @@
                 'mezzanine.template.loader_tags',
             ],
             'libraries': {
-                'features_adminsortable_tags': 'networkapi.features.templatetags'
-                                               '.features_adminsortable_tags_custom'
+                'adminsortable_tags':
```

Comment:
This literally needs to be named `adminsortable_tags` because that name is what is used throughout the adminsortable default templates that we do not override.